### ISSUE TITLE
Off in production

### ISSUE THREAD

**Issue by jonabaptistella** [jenstornell/panel-brand#2]
I miss one option to turn it completely off in production. Or is it one option like that...?

**Comment by jenstornell** [comment]
No, I've missed that option but it's a good suggestion. As much as I would love it to be in the plugin I really hope that the Kirby crew will make it built in. Else that work will be on every plugin developer out there.

**But so far I'm out of luck:**

https://forum.getkirby.com/t/disable-plugins-by-config/3921
https://forum.getkirby.com/t/official-plugin-enable-disable-option-by-the-plugin-authors/5905/9

The crew does not agree with me here. I guess they see the world a bit different in this case.

I'll keep it open until I will add an option. I can't promise you when.

**Comment by jonabaptistella** [comment]
This is maybe a little wrong based on my PHP knowledge but isn´t that just to wrap the plugin in a boolean? Or are I mistaken?

**Comment by jenstornell** [comment]
Yes, it would be something like:

```php
if( c::get('plugin.panel.brand.enable', true) === true ) {
  // Run the plugin
}
```

Skype? What do you want to chat about?

**Comment by jonabaptistella** [comment]
@jenstornell I wanted have a quick chat about if it is possible to work together... etc...

**Comment by jonabaptistella** [comment]
Since I can not find your email anywhere, please shoot me an email and we can take it from there... I do need someone with more back-end dev. skills then me... Hope you will like a chat what we can do together...
